modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= Supported platforms for managed clusters
```

Comment:
Should the title begin with a gerund phrase because it is a procedure module?

Reply:
Yes, great catch! Updated as suggested.